### ISSUE TITLE
Default gateway monitoring

### ISSUE THREAD

**Issue by strekm** [kyma-project/api-gateway#597]
**Description**

Create Istio VirtualService exposing healthz enpoint of Istio when default gateway is created. VS should be removed when default gateway is removed.

AC:
- [x] default gateway monitoring

**Reasons**

**DoD:**
~~- [ ] Create a follow-up issue.~~
- [x] Provide unit tests.
- [x] Provide integration tests.
- [x] Provide documentation.
- [x] Test on a production-like environment.
- [x] Check the outcome of all related pipelines.
~~- [ ] Verify resource limits.~~
- [x] As a PR reviewer, verify code coverage and evaluate if it is acceptable.
~~- [ ] Create a release and bump in Kyma.~~
~~- [ ] Add release notes and What's New notes for Kyma customers.~~

**Attachments**
part of: https://github.com/kyma-project/api-gateway/issues/130
depends on: #532 , #560
